pkgs/build-support/node/import-npm-lock/default.nix

```diff
@@ -127,8 +127,54 @@ in
       cp "$packageLockPath" $out/package-lock.json
     '';
 
+  # Build node modules from package.json & package-lock.json
+  buildNodeModules =
+    let
+      cleanAttrs = lib.flip removeAttrs [ "npmRoot" "package" "packageLock" "nodejs" ];
```

Comment:
Could you add a comment above this explaining why these are omitted?

Reply:
With `derivationArgs` cleaning is no longer required.